akka-docs/src/test/java/jdocs/actor/ActorDocTest.java

```diff
@@ -585,6 +585,37 @@ public void using_receiveTimeout() {
     };
   }
 
+  static class ReceiveTimeoutWithCancelActor extends AbstractActor {
+    public ReceiveTimeoutWithCancelActor() {
+      getContext().setReceiveTimeout(Duration.ofSeconds(1));
+    }
+
+    @Override
+    public Receive createReceive() {
+      return receiveBuilder()
+          .matchEquals(
+              "Hello",
+              s -> {
+                // To set in a response to a message
+                getContext().setReceiveTimeout(Duration.ofMillis(200));
```

Comment:
Shouldn't it (also) cancel the timeout, since that is what we are testing?

Reply:
oops, I was buying stocks when coding this... will update then